src/sentry/models/groupsearchviewstarred.py

```diff
@@ -8,10 +8,27 @@
 from sentry.db.models.base import DefaultFieldsModel
 from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
 from sentry.db.models.manager.base import BaseManager
+from sentry.models.groupsearchview import GroupSearchView
 from sentry.models.organization import Organization
 
 
 class GroupSearchViewStarredManager(BaseManager["GroupSearchViewStarred"]):
+    def num_starred_views(self, organization: Organization, user_id: int) -> int:
+        """
+        Returns the number of starred views for a user in an organization.
+        """
+        return self.filter(organization=organization, user_id=user_id).count()
+
+    def get_starred_view(
+        self, organization: Organization, user_id: int, view: GroupSearchView
+    ) -> "GroupSearchViewStarred | None":
```

Comment:
if you put `from __future__ import annotations` at top of file i don't think you need the quotes around the return types?